src/Cpp/1-getting-started/1-HelloWindow/HelloWindow.cpp

```diff
@@ -1,18 +1,13 @@
 #include "HelloWindow.hpp"
 #include "Input/Input.hpp"
 
+#include <GLFW/glfw3.h>
 #include <iostream>
 #include <string>
-#include <GLFW/glfw3.h>
 
-HelloWindowApplication::HelloWindowApplication(const std::string_view title)
-    : Application(title)
-{
-}
+HelloWindowApplication::HelloWindowApplication(const std::string_view title) : Application(title) {}
```

Comment:
When calling base impls is it possible to keep it on the next line indendet 4 spaces in?